homeassistant/components/insteon_plm.py

```diff
@@ -165,10 +256,12 @@ def __getitem__(self, key):
 class InsteonPLMEntity(Entity):
     """INSTEON abstract base entity."""
 
-    def __init__(self, device, state_key):
+    def __init__(self, hass, device, state_key):
         """Initialize the INSTEON PLM binary sensor."""
+        self._hass = hass
```

Comment:
This is not necessary. All entities that are added to Home Assistant will have access to the `hass` object via `self.hass`